editor/neovim/tree-sitter-slint/binding.gyp

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Should we have (some|all) of these files in editors/tree-sitter? The parser itself can be used for more than neovim, can't it?

Reply:
Can you add a license header into the file?

Reply:
> Should we have (some|all) of these files in editors/tree-sitter? The parser itself can be used for more than neovim, can't it?

Yes. So that would make more sense


> Can you add a license header into the file?

Yep. I'll get that done.